apps/trading/pages/client-router.tsx

```diff
@@ -104,6 +116,8 @@ const routerConfig: RouteObject[] = [
     element: <LazyDisclaimer />,
   },
   { path: Routes.DEPOSIT, element: <LazyDeposit /> },
+  { path: Routes.WITHDRAW, element: <LazyWithdraw /> },
```

Comment:
(non blocking): We should probably wrap these with a layout route to ensure consistency

Reply:
I'll make an additional PR for this